README.md

```diff
@@ -1,2 +1,9 @@
 # pyhf-benchmark
-Benchmarking of hardware acceleration of pyhf
+
+Benchmarking of hardware acceleration of `pyhf`
+
+## Authors
+
+`pyhf-benchmark` is openly developed by [Bo Zheng](https://iris-hep.org/fellows/BoZheng.html) and [Matthew Feickert](https://www.matthewfeickert.com/).
```

Comment:
```suggestion
`pyhf-benchmark` is openly developed by [Bo Zheng](https://iris-hep.org/fellows/BoZheng.html) and the [`pyhf` dev team](https://scikit-hep.org/pyhf/#authors).
```

I think we can put the focus on you and then just mention that we're also involved.

Reply:
Done!